local/seed-bdrs.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 Volkswagen AG
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+KEY=
```

Comment:
I would suggest to remove this file. 
Instead add some code to the generatey-keys.sh, in order to have it auto-generated (just like we do with the .env file). 

This would greatly help us developers to avoid the tedious copying of the EDC Key to this file here.

Reply:
incorporated in branch chore/edc-0.7.0-infra

Reply:
done